types/web3/web3-tests.ts

```diff
@@ -1,19 +1,28 @@
+// --------------------------------------------------------------------------
+// Many type definitions do not correct at this momonet
+// Please contribute to add more tests to check function parameters and return type
+// Please use the official docs(https://web3js.readthedocs.io/en/1.0/#) and/or
+// the code(v1.0.0-beta.36(https://github.com/ethereum/web3.js/tree/v1.0.0-beta.36)
+// as a reference until all type definitions are correct
+// --------------------------------------------------------------------------
 import Web3 = require("web3");
 import BigNumber = require("bn.js");
 
 const contractAddress = "0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe";
+const privateKey = "0x348ce564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709";
 
 //
 // Web3
 // --------------------------------------------------------------------------
 const web3 = new Web3();
 const myProvider = new web3.providers.HttpProvider("http://localhost:5454");
-web3.setProvider(myProvider);
-web3.eth.setProvider(myProvider);
```

Comment:
You removed these test but I cannot see where they have been readded. I think they should stay, right?

Reply:
Will you fix them?